imports/plugins/core/orders/client/components/OrderCardFulfillmentGroupStatusButton.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Mutation } from "react-apollo";
+import withStyles from "@material-ui/core/styles/withStyles";
+import FormControl from "@material-ui/core/FormControl";
+import Grid from "@material-ui/core/Grid";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import OutlinedInput from "@material-ui/core/OutlinedInput";
+import Select from "@material-ui/core/Select";
+import { i18next, Reaction } from "/client/api";
+import ConfirmButton from "/imports/client/ui/components/ConfirmButton";
+import updateOrderFulfillmentGroupMutation from "../graphql/mutations/updateOrderFulfillmentGroup";
+
+const styles = (theme) => ({
+  formControl: {
+    margin: theme.spacing.unit,
+    minWidth: 120
```

Comment:
We should add a minWidth to the MUI theme, and use it across all dialogs for consistence.

Reply:
This is a min-width for that specific form field, based on the data inside the form, not for the dialog itself.